Tests/TuistCacheTests/ContentHashing/CoreDataModelsContentHasherTests.swift

```diff
@@ -0,0 +1,101 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+
+final class CoreDataModelsContentHasherTests: TuistUnitTestCase {
+    private var subject: CoreDataModelsContentHasher!
+    private var coreDataModel: CoreDataModel!
+    private var temporaryDirectory: TemporaryDirectory!
```

Comment:
You don't need to have an attribute for this. Just call [this method](https://github.com/tuist/tuist/blob/master/Sources/TuistSupportTesting/TestCase/TuistTestCase.swift#L64) from your test function and it'll give you a temporary directory that you can use for running your test. Once the test is finished, the directory is deleted automatically.

Reply:
👍 thanks